src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,828 @@
+---
+title: Hello Darkness, My Old Friend
+subhead: Overhyped or necessity? Learn everything about dark mode and how to support it to benefit your users!
+authors:
+  - thomassteiner
+date: 2019-06-25
+hero: hero.jpg
```

Comment:
Do you have a high res version of this image?
I think we need it at 1600 x 800 and a 2x version. We haven't been using srcset for these images but we should start so having the 2x version handy would be good.

Reply:
I have just added the 1600x800 version in https://github.com/GoogleChrome/web.dev/pull/1021/commits/e82cdb6493ed94a7523f3dd4de57cb6c411e2c19.